cmd/vulcan-aws-alerts/main.go

```diff
@@ -31,9 +32,17 @@ func main() {
 		if vulcanAssumeRoleEndpoint == "" {
 			return fmt.Errorf("VULCAN_ASSUME_ROLE_ENDPOINT option is missing")
 		}
-
 		roleName := os.Getenv("ROLE_NAME")
 
+		isReachable, err := helpers.IsReachable(target, assetType,
+			helpers.NewAWSCreds(vulcanAssumeRoleEndpoint, roleName))
+		if err != nil {
+			logger.Warnf("Can not check asset reachability: %v", err)
+		}
+		if !isReachable {
+			return checkstate.ErrAssetUnreachable
```

Comment:
In relation with the discussion we have in https://github.com/adevinta/vulcan-check-sdk/pull/13. Is this retro-compatible?

Because another option would be that `helpers.IsReachable` returns always `true` when the asset type is not provided, instead of falling back to asset type auto-detection.

Reply:
Agree. See [SDK PR discussion](https://github.com/adevinta/vulcan-check-sdk/pull/13#discussion_r518809929).